endpoints/api/test/test_security.py

```
@@ -4185,6 +4185,225 @@
         "reader",
         403,
     ),
+    # RepositoryTagImmutability - public/publicrepo
+    (
+        RepositoryTagImmutability,
```

Ruff (0.14.11)

`RepositoryTagImmutability` may be undefined, or defined from star imports

(F405)